cinema/serializers.py

```diff
@@ -106,15 +132,15 @@ class Meta:
 
 
 class MovieSessionDetailSerializer(MovieSessionSerializer):
-    movie = MovieListSerializer(many=False, read_only=True)
+    image = MovieImageSerializer(allow_null=True)
     cinema_hall = CinemaHallSerializer(many=False, read_only=True)
     taken_places = TicketSeatsSerializer(
         source="tickets", many=True, read_only=True
     )
 
     class Meta:
         model = MovieSession
-        fields = ("id", "show_time", "movie", "cinema_hall", "taken_places")
+        fields = ("id", "show_time", "image", "cinema_hall", "taken_places")
```

Comment:
Image url should be shown on detail page inside movie -> image. change image to movie